src/app/components/MediaLoader/utils/buildSettings.test.ts

```diff
@@ -1111,5 +1121,100 @@ describe('buildSettings', () => {
         showAds: false,
       });
     });
+
+    it('Should process a live media block with service ID into a valid playlist item', () => {
+      const mediaBlock = {
+        type: 'liveMedia',
+        model: {
+          urn: 'urn:bbc:pips:sid:bbc_arabic_tv',
+          title: 'BBC Arabic TV',
+          type: 'episode',
+          synopses: {
+            short:
+              'يلقي برنامج إكسترا التلفزيوني الضوء على قصص إنسانية فريدة من حياتنا، نعيشها ونرويها للشباب العربي',
+            medium:
+              'يلقي برنامج إكسترا التلفزيوني الضوء على قصص إنسانية فريدة من حياتنا، نعيشها ونرويها للشباب العربي',
+            long: 'يلقي برنامج إكسترا التلفزيوني الضوء على قصص إنسانية فريدة من حياتنا، نعيشها ونرويها للشباب العربي',
+          },
+          mediaType: 'audio_video',
+          imageUrlTemplate:
+            'https://ichef.bbci.co.uk/images/ic/$recipe/p0hwr92l.jpg',
+          masterbrand: {
+            id: 'bbc_arabic_tv',
+            name: 'تلفزيون بي بي سي عربي',
+            networkName: 'تلفزيون بي بي سي عربي',
+            type: 'tv',
+            imageUrlTemplate: 'ichef.bbci.co.uk/images/ic/$recipe/p08b23t4.png',
+          },
+          version: {
+            vpid: 'n4pdlz1s10',
+            duration: 'PT30M',
```

Comment:
These values appear to be the most recent programme which was broadcast on the live channel.